en/integrations/event-management/sc-bsm.md

```diff
@@ -3,124 +3,255 @@ id: sc-hp-bsm
+
+**Add** a new **generic - stream connector** output and **set** the following fields as follow:
+
+| Field           | Value                                                  |
+| --------------- | ------------------------------------------------------ |
+| Name            | BSM                                                    |
+| Path            | /usr/share/centreon-broker/lua/bsm_connector-apiv2.lua |
+
+### Add BSM optional parameters
+
+Some stream connectors have a set of optional parameters dedicated to the Software that they are associated with. To add them you must **click** on the **+Add a new entry** button located **below** the **filter category** input.
+
+| Type   | Name                | Value (explanation)                                                                                                    | defaultvalue                                                      |
+|--------|---------------------|------------------------------------------------------------------------------------------------------------------------| ----------------------------------------------------------------- |
+| String | `http_server_url`   | URL de votre plateforme BSM                                                                                            | `https://<my.bsm.server>:30005/bsmc/rest/events/<my-webservice>/` |
+| String | `http_proxy_string` | Paramétrage du proxy permettant de sortir vers Internet en HTTP/HTTPS                                                  | `http://your.proxy.server:3128`                                   |
+| String | `source_ci`         | Nom permettant d'identifier l'émetteur                                                                                 | `Centreon`                                                        |
+| Number | `log_level`         | Niveau de verbosité des logs 0: errors seulement, 1: +warnings, 2: +verbose, 3: +debug                                 | 2                                                                 |
+| String | `log_path`          | Chemin complet du fichier de log                                                                                       | `/var/log/centreon-broker/my-custom-logfile.log`                  |
+| Number | `max_buffer_size`   | Nombre maximum d'événements à stocker en mémoire tampon en attendant de les transmettre en un seul envoi               | 1                                                                 |
+| Number | `max_buffer_age`    | Temps d'attente maximum avant d'envoyer les événements en mémoire tampon si `max_buffer_size` n'est pas encore atteint | 5                                                                 |
```

Comment:
you need to split this part into two chapters. One about mandatory parameters. and one about optional parameters. Those two chapters must only talk about BSM dedicated parameters. Others such as max_buffer_xxx  logs and so on must be explained in another chapter

please, use only english